### PR TITLE
update documentation after move to openmass

### PR DESCRIPTION
**Pull request by lmirabile**
**Description:**
Explain the technical implementation of the work done.

**Jira:**
https://jira.mass.gov/browse/DP-****

**To Test:**
- [ ] Add steps to test this feature

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/mass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Review by weitzman — APPROVED** [review]
There are more changes needed in /docs folder but you probably know tat. They can be followups as desired

File: README.md [hunk 1]
```diff
@@ -2,15 +2,13 @@
 
 _The official website of the Commonwealth of Massachusetts_
 
-This is the codebase for the Drupal 8 www.mass.gov. The site's theme (mass_theme) is powered by Mayflower, a companion repo available at https://github.com/massgov/mayflower.
-
-[Table of Contents](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/README.md) for documentation related to this repository such as [Releases](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/release.md). NOTE: documentation specific to the production site is not publicly available. 
+This is the codebase for the Drupal 8 web site `www.mass.gov`. The site's theme [mass_theme](https://github.com/massgov/openmass/blob/develop/docroot/themes/custom/mass_theme/README.md) is powered by Mayflower, a companion repo available at https://github.com/massgov/mayflower.
```
Comment (mrossi113) [inline README.md]: The release link has been removed? I thought we were keeping a link to the release for the release manager.

**Review by mrossi113 — COMMENTED** [review]
@lmirabile I have one question.

File: README.md (hunk 1 above)
Comment (lmirabile) [reply README.md]: Oops, didn't mean to do that. Will add it back, thanks.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
-This is the codebase for the Drupal 8 www.mass.gov. The site's theme (mass_theme) is powered by Mayflower, a companion repo available at https://github.com/massgov/mayflower.
+This is the codebase for the Drupal 8 web site `www.mass.gov`. The site's theme, [mass_theme](https://github.com/massgov/openmass/blob/develop/docroot/themes/custom/mass_theme/README.md), is powered by Mayflower, a companion repo available at https://github.com/massgov/mayflower.
 
-[Table of Contents](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/README.md) for documentation related to this repository such as [Releases](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/release.md). NOTE: documentation specific to the production site is not publicly available. 
+See the [Table of Contents](docs/README.md) for additional documentation related to this repository. In addition, there is [documentation for the Mass Digital team](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/README.md) that covers topics such as [Releases](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/release.md).
```
Comment (lmirabile) [inline README.md]:
```suggestion
See the [Table of Contents](/docs/README.md) for additional documentation related to this repository. In addition, there is [documentation for the Mass Digital team](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/README.md) that covers topics such as [Releases](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/release.md). 
```

**Pull request merged**